include/revng/ADT/GenericGraph.h

```diff
@@ -226,6 +227,9 @@ class ForwardNode
   ForwardNode(const ForwardNode &) = default;
   ForwardNode(ForwardNode &&) = default;
 
+  NodeData copy() const { return *this; };
+  NodeData &&move() { return std::move(*this); };
```

Comment:
These methods might be a bit suspicious in the context of the class that inherits from it.
What about `copyData`/`moveData`?

Reply:
Done.